server/server.ts

```diff
@@ -47,6 +47,36 @@ app.get('/isAngularProject', (req, res) => {
   );
 });
 
+app.get('/projects', (req, res) => {
+  const projects: string[] = [];
+  const subDirs = fs.readdirSync(process.cwd());
```

Comment:
`subDirs` is an array that holds directories AND files.
I think that something like `folderContent` is preferable.